apps/omg_db/lib/omg_db/measure.ex

```diff
@@ -19,7 +19,7 @@ defmodule OMG.DB.Measure do
   alias OMG.Status.Metric.Datadog
   import OMG.Status.Metric.Event, only: [name: 1]
 
-  alias OMG.DB.RocksDB.Server
+  alias RocksDB.Server
```

Comment:
moving this out of the namespace makes sense, but should the file path also match?